tests/python/chroma_downsample_image/requirements.txt

```diff
@@ -1,3 +1,3 @@
 numpy
-opencv-python
+opencv-python==
```

Comment:
What does == without a rhs? O.o

Reply:
Oh. Oops!! Nothing to see here. (Long story but this change is wrong and shouldn't be here)